README.md

```diff
@@ -1,2 +1,11 @@
 # im-nit
 Discord bot
+
+## Database
+This bot uses a database. In order to set it up, you should follow this steps:
+<ol>
```

Comment:
Why not use the markdown format for ordered lists? 
https://www.markdownguide.org/basic-syntax/#ordered-lists